frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import {
+  Rule,
+  Alert,
+  StateCounts,
+  Severity,
+  AlertState,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import { alertDescription, alertState } from '@console/internal/reducers/monitoring';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+enum AlertStates {
```

Comment:
Should use the existing `AlertStates` enum instead of creating a new one here.

Reply:
I see there appears to be a difference in string case between the states from Prometheus and Thanos, but since we are converting to lower case anyway for comparing, I still think we should use the existing `AlertStates` enum instead.

Reply:
Done